package.json

```diff
@@ -179,6 +180,7 @@
     "react-remove-scroll": "^2.4.0",
     "react-router-dom": "5.2.0",
     "react-select": "3.1.0",
+    "react-table": "^7.6.0",
```

Comment:
Remove hat I guess?

Also, I'll defer the approval of this package to @arnifreyrs